jx_sqlite/aggs_table.py

```diff
@@ -333,6 +333,26 @@ def _get(row):
                     column_alias=quote_table(s.name),
                     type=sql_type_to_json_type["n"]
                 )
+            elif s.aggregate == "count" and (not query.edges and not query.groupby):
+                value=s.value.var
+                columns=[c.es_column for c in self.sf.columns if untyped_column(c.es_column)[0]==value]
+                if s.default != None:
```

Comment:
the `default` option is only used if the COUNT is NULL, since COUNT is never NULL, it is never needed; we do not need this check.